tests/models/test_model.py

```diff
@@ -154,7 +154,6 @@ def test_model_log():
 
         loaded_model = Model.load(os.path.join(local_path, "MLmodel"))
         assert loaded_model.run_id == r.info.run_id
-        assert loaded_model.artifact_path == "some/path"
```

Comment:
`artifact_path` is no longer `some/path`. Not sure if it makes sense to have `artifact_path` in MLflow 3.0.